### PR TITLE
Update dependencies for various subprojects

### PR DESCRIPTION
**Pull request by ehoogerbeets**

### CODE REVIEW

File: android/project.json [hunk 1]
```diff
@@ -16,7 +16,9 @@
     ],
     "excludes": [
         ".*",
-        "xliffs"
+        "xliffs",
+    	"res",
+    	"node_modules"
```
Comment (gouniLee) [inline android/project.json:21]: node_modules is already added in exclude list from loctool. no need to write additionally.

Comment (ehoogerbeets) [reply android/project.json]: fixed

File: js/project.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
         "xliffs",
         "resources",
         "package.json",
-        "package-lock.json"
+        "package-lock.json",
+    	"node_modules"
```
Comment (gouniLee) [inline js/project.json:22]: package-lock.json and package.json are also already in the excluded list from loctool.
so It's okay to remove package.json, package-lock.json, and node_modules.

File: markdown/project.json [hunk 3]
```diff
@@ -9,7 +9,8 @@
     ],
     "excludes": [
         "xliffs",
-        "locale"
+        "locale",
+    	"node_modules"
```
Comment (gouniLee) [inline markdown/project.json:13]: same here: 'node_modules`

File: webos-js/project.json [hunk 4]
```diff
@@ -18,7 +18,8 @@
         ".*",
         "test",
         "xliffs",
-        "common"
+        "common",
+    	"node_modules"
```
Comment (gouniLee) [inline webos-js/project.json:22]: here too.

File: js/project.json (hunk 2 above)
Comment (ehoogerbeets) [reply js/project.json]: fixed

File: markdown/project.json (hunk 3 above)
Comment (ehoogerbeets) [reply markdown/project.json]: fixed

File: webos-js/project.json (hunk 4 above)
Comment (ehoogerbeets) [reply webos-js/project.json]: fixed

**Comment by ehoogerbeets** [comment]
Ready to re-review

**Review by gouniLee — APPROVED** [review]

**Pull request merged**